src/script/common/c_packer.cpp

```diff
@@ -98,6 +98,28 @@ static inline bool suitable_key(lua_State *L, int idx)
 	}
 }
 
+/**
+ * Push core.known_metatables to the stack if it exists.
+ * @param L Lua state
+ * @return true if core.known_metatables exists, false otherwise.
+*/
+static inline bool get_known_lua_metatables(lua_State *L)
+{
+	lua_getglobal(L, "core");
+	if (!lua_istable(L, -1)) {
+		lua_pop(L, 1);
+		return false;
+	}
+	lua_getfield(L, -1, "known_metatables");
+	if (lua_istable(L, -1)) {
+		lua_insert(L, -2);
+		lua_pop(L, 1);
```

Comment:
if I understood correctly this does the same and is clearer IMO
```suggestion
		lua_remove(L, -2);
```